src/io/fastq.rs

```diff
@@ -82,25 +96,41 @@ impl<R: io::Read> Reader<R> {
 /// A FastQ record.
 #[derive(Debug, Clone, Default)]
 pub struct Record {
-    header: String,
-    seq: String,
-    qual: String,
+    pub id: String,
+    pub desc: String,
```

Comment:
I'd prefer these attributes to not be public. Is there a specific reason for making them public?